docs/pages/blog/aggregation-functions.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Aggregate data like in Excel, but easier!
+description: Introducing aggregation functions and summary rows in Data grid Premium.
+date: 2022-07-31T00:00:00.000Z
+authors: ['josefreitas', 'flaviendelangle', 'cherniavskii']
+tags: ['MUI X', 'News']
+card: true
+---
+
+If you ever had a lot of data on a grid, you probably know that the ability to set different perspectives on data is essential to help users extract information. Typically, one would use at least filtering or sorting, but that barely scratches the surface of what one could do.
+
+The Data grid Premium is all about enabling and empowering advanced use cases for data analysis.
+And in that spirit, we’re overjoyed to announce that starting from [v5.15.0](https://github.com/mui/mui-x/blob/master/CHANGELOG.md#5150), you can now use [aggregation functions](/x/react-data-grid/aggregation/), and with a couple of clicks, extract information like **sum**, **average**, **count**, and others.
+
+## Wait, what is an aggregation function?
+
+In a nutshell, those are functions used to return a computed result based on the values of a given column. For example, you have a list of financial transactions, and you want to get the Sum of all transactions and display it in a summary row.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/aggregation-functions/summary-row.mp4" type="video/mp4" />
+</video>
+
+## High end-user experience
+
+The **Sum** function showing a `Total` in summary rows is probably one of the most common use cases. But users can employ multiple aggregation functions in different columns simultaneously, and even combine with row grouping to aggregate children’s data on each group.
+
+To make all that work well together, we took to the heart that we’d need to bring and focus on the best end-user experience possible, enabling the majority of use cases out of the box to the end-user, without any significant need for customization.
+
+How easy can it be to get the average of transactions per user?
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
+  <source src="/static/blog/aggregation-functions/with-row-grouping.mp4" type="video/mp4" />
+</video>
+
+## Customize everything
+
+The out-of-the-box experience is very important to us, but as is the ability to customize and extend the feature to your own use case.
+
+You can change the styles of every element involved, set automatic aggregations, choose which columns can be aggregated by which functions, and even create your own custom function, like in the example below, where we created a function to get the first value in alphabetical order.
+
+```tsx
+const firstAlphabeticalAggregation: GridAggregationFunction<string, string | null> =
```

Comment:
Could we prettify this demo so there is no horizontal scrollbar?

<img width="711" alt="Screenshot 2022-07-25 at 18 08 18" src="https://user-images.githubusercontent.com/3165635/180824615-38161f67-b7a5-471d-9a63-e1b9115d4691.png">

Reply:
Now, to make this happen, we would need:

```diff
diff --git a/docs/pages/blog/mui-x-v5.md b/docs/pages/blog/mui-x-v5.md
index 151a906b27..ef430c0146 100644
--- a/docs/pages/blog/mui-x-v5.md
+++ b/docs/pages/blog/mui-x-v5.md
@@ -129,7 +129,9 @@ const GridToolbarContainerStyled = styled(GridToolbarContainer)({
 });

 function MyCustomToolbar() {
-  return <GridToolbarContainerStyled>My custom toolbar</GridToolbarContainerStyled>;
+  return (
+    <GridToolbarContainerStyled>My custom toolbar</GridToolbarContainerStyled>
+  );
 }

 export default function App() {
diff --git a/prettier.config.js b/prettier.config.js
index 44d221920b..47f13bd576 100644
--- a/prettier.config.js
+++ b/prettier.config.js
@@ -10,5 +10,12 @@ module.exports = {
         printWidth: 85,
       },
     },
+    {
+      files: ['docs/pages/blog/**/*.md'],
+      options: {
+        // otherwise code blocks overflow on the docs website
+        printWidth: 82,
+      },
+    },
   ],
 };
```

Reply:
Thanks for diving there!
I'll copy your prettified version from this PR. We can open a new one for the prettier.config update.

Reply:
PR to address prettier config update: https://github.com/mui/material-ui/pull/33648